src/js/hooks/reporting/useReorderReportFilters.js

```diff
@@ -109,7 +109,8 @@ const useReorderReportFilters = () => {
     ]);
     setCategories(categoryList);
     setTags(tagList);
-    setLocations(locationList);
+    // Filter out current location from additional inventory locations
+    setLocations(locationList.filter((location) => location.id !== currentLocation?.id));
```

Comment:
I know ideally this should have been done on the backend, but since the locations API is already overwhelming, I decided not to add another piece of code to that.